### PR TITLE
FreeBSD little build fix proposal.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Review by refi64 — APPROVED** [review]
One minor nit but LGTM overall, thanks!!

File: tsc/mruby_tsc_build_config.rb [hunk 1]
```diff
@@ -43,7 +43,11 @@
 end
 
 MRuby::Build.new do |conf|
-  toolchain :gcc
+  if RUBY_PLATFORM !~ /freebsd/
```
Comment (refi64) [inline tsc/mruby_tsc_build_config.rb]: Nit: while we're doing this anyway, maybe also check for openbsd here?

**Comment by Quintus** [comment]
Very nice. Thank you. I approve this, but it would be nice if someone could check if compiling works afterwards still on MacOS.

**Comment by Quintus** [comment]
@refi64 Consider cherry-picking this into the release branch for 2.1.0. It should apply nearly as-is and does not change anything that could possible make problems I think.

**Comment by devnexen** [comment]
Side note: would be nice to update the mruby dependency at some point. this https://github.com/mruby/mruby/blob/1c09046c13fc0a763583c6da0f1d350f6c41c4ca/mrbgems/mruby-compiler/core/keywords probe signatures are not correct (unsigned int instead of size_t) and has been fixed since.

**Comment by Quintus** [comment]
Thanks for the heads-up. I will take care of mruby updates in the `devel` branch later on. The `release-2.1.0` branch should not see such changes anymore.

**Pull request merged**